custom-plugins/fictive_studios/admin/printing-area/api/print-area-api.js

```diff
@@ -1,13 +1,17 @@
-document
-  .getElementById("getPrintArea")
-  .addEventListener("click", getPrintAreas);
+class PrintAreaApi {
+  constructor() {}
 
-function getPrintAreas() {
-  var xhr = new XMLHttpRequest();
-  var params = new URLSearchParams();
-  params.append("action", "get_print_areas");
-  var url = ajaxurl + "?" + params.toString();
-  xhr.open("GET", url, true);
-  xhr.onload = function () {};
-  xhr.send();
+  getPrintAreas() {
```

Comment:
Remove this.

Reply:
I think you can remove the whole file.

Reply:
Don't you need the api for builder?

Reply:
yes.. but this code is not needed here..